tests/smoke/load_balancer.bats

```diff
@@ -70,14 +70,6 @@ certificate_name="l0-$LAYER0_INSTANCE-api"
     l0 loadbalancer create --port 80:80/http test loadbalancer4
 }
 
-@test "deploy create guestbook" {
```

Comment:
Did you remove these because they aren't loadbalancer-specific and/or there is overlap with `service` smoketests?

Reply:
Both. This also allows the loadbalancer tests to run w/o the need for ec2 instances, taking the runtime from ~5 minutes to < 1.